### PR TITLE
added aria-hidden for Tabs empty divs

### PR DESCRIPTION
**Pull request by zhukovka**
https://jira.wixpress.com/browse/GROUP-2601

### CODE REVIEW

**Comment by jonathanadler** [comment]
@zhukovka 
In order for the build to run, you'll have to sign your commit
https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md

Sorry for the hassle, but it's necessary 🙏

File: src/components/Tabs/Tab/Tab.tsx [hunk 1]
```diff
@@ -51,8 +51,8 @@ export const Tab = React.forwardRef<HTMLLIElement, TabProps>((props, ref) => {
       aria-current={isActive}
     >
       <span role="link">{title}</span>
-      <div className={classes.focusIndicatorWrapper} tabIndex={-1}>
-        <div className={classes.focusIndicator} tabIndex={-1} />
+      <div className={classes.focusIndicatorWrapper} tabIndex={-1} aria-hidden={'true'}>
```
Comment (jonathanadler) [inline src/components/Tabs/Tab/Tab.tsx:54]: The parent `aria-hidden` is enough, since it affects its children as well
So no need to have both

**Review by jonathanadler — APPROVED** [review]
Just see my comment

File: src/components/Tabs/Tab/Tab.tsx (hunk 1 above)
Comment (zhukovka) [reply src/components/Tabs/Tab/Tab.tsx]: I removed the descendant aria-hidden and I signed the last commit. Is it enough for signing?

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_444/

**Pull request merged**